### PR TITLE
Create Testskript.R

### LINKED ISSUE #2: Erstellt ein R Skript

### PR DESCRIPTION
**Pull request by ruckeran**
close #2

### CODE REVIEW

File: Testskript.R [hunk 1]
```diff
@@ -0,0 +1 @@
+# Hallo Jule, das ist unser Testskript :-)
```
Comment (burbliesj) [inline Testskript.R]: Awesome, you did amazing!
But please add one more smiley! ;)

**Review by burbliesj — CHANGES_REQUESTED** [review]
Looks great, but more smiley faces please!

File: Testskript.R (hunk 1 above)
Comment (ruckeran) [reply Testskript.R]: Good idea, thanks for your input!

**Review by burbliesj — APPROVED** [review]
Enough smileys, thanks!

**Pull request merged**